README.md

```diff
@@ -11,3 +11,30 @@ is potentially incompatible with Trusted Types, tsec emits compilation errors.
 
 tsec is based on the open source TypeScript static analyzer
 [Tsetse](https://tsetse.info/).
+
+## Build and run
+
+### Build
+```
+  # Clone the repository
+  git clone https://github.com/googleinterns/tsec
+  cd tsec
+  
+  # Install gulp
+  npm install gulp
+  
+  # Install dependencies
+  npm install
+  
+  # Run a compile based on tsconfig.json
+  node_modules/typescript/bin/tsc 
+  
+```
+
+### Run
+
+To run a check for the compatibility with Trusted Types in the project containing `tsconfig.json` use:
+```
+  {PATH_TO_TSEC}/bin/tsec -p tsconfig.json
+```
+Add `--no-Emit` flag to skip the output from compilation.
```

Comment:
What are we skipping here? If I run `{PATH_TO_TSEC}/bin/tsec -p tsconfig.json` it works the same with or without a flag for me.

Reply:
Hmm, the flag should be `--noEmit`